packages/transport/package.json

```diff
@@ -44,6 +44,7 @@
         "prepublish": "yarn tsx ../../scripts/prepublish.js"
     },
     "devDependencies": {
+        "@trezor/react-native-usb": "workspace:*",
```

Comment:
the other option would be creating separate package 

@trezor/transport-native

that would define `packages/transport/src/transports/nativeUsb.ts` using imports from `@trezor/transport`

Reply:
That makes sense, but not sure if there is any benefit because it must be imported in connect anyway to register so it will be in connect dependency tree anyway.

Only way arround is some dynamic transport injecting during connect init.